hue/hue.sh

```diff
@@ -41,6 +41,10 @@ function retry_apt_command() {
 }
 
 function update_apt_get() {
+  if grep ubuntu /etc/apt/sources.list > /dev/null 2>&1; then
```

Comment:
I do not think that we should do this because we do not do this in other init actions.

Also, we usually check OS like this:
```bash
if [[ "$(. /etc/os-release && echo "${ID}")" == 'ubuntu' ]]; then
```